src/sdgio-edit.owl

```diff
@@ -1515,6 +1519,24 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
 AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020232> "system level outcome"@en)
 SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020232> <http://purl.unep.org/sdg/SDGIO_00000251>)
 
+# Class: <http://purl.unep.org/sdg/SDGIO_00020236> (intermediate development outcome)
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020236> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020236> "2019-04-28T11:19:21Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.unep.org/sdg/SDGIO_00020236> "IDO")
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://hdl.handle.net/10947/3865>) rdfs:comment <http://purl.unep.org/sdg/SDGIO_00020236> "A CGIAR objective specification which 1) names an objective the achievement of which contributes to the achievement of a specific system level outcome and 2) indicates to which system level outcome a piece of CGIAR research contributes and how such a piece of research contributes to that system level outcome.")
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020236> "intermediate development outcome"@en)
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020236> <http://purl.unep.org/sdg/SDGIO_00000251>)
+
+# Class: <http://purl.unep.org/sdg/SDGIO_00020237> (sub-intermediate development outcome)
```

Comment:
The label here is a bit off, I think. 
It should be "sub-outcome of an intermediate development outcome"

Reply:
I understand that my label sounds odd (to be gentle). However, this is exactly what CGIAR people say. I used this piece of text: "Below this level [i.e. IDOs] are Sub-Intermediate Development Outcome (sub-IDOs)" (p. 14 of the document _CGIAR Strategy and Results Framework)_. 

It may be relevant to note that the language of the document is sometimes not that great. Do you think that the need for a decently-formulated label is more important than faithfulness to the source of that label? How do you reason in cases like this one?

Reply:
If we want to stick with the CGIAR language, it would be better to keep the main labels as they are stated in the SRF document. The label that @pbuttigieg proposed, which is going to be more understandable by external people, can be added as exact synonym. What do you think?

Reply:
Following Pier Luigi's suggestion, repeated in our Skype conversation on 29 Apr 2019, I used "sub-outcome of an intermediate development outcome" as the label, for the sake of linguistic correctness. I added "sub-intermediate development outcome" and "sub-IDO" as two distinct hasExactSynonym.

Reply:
There's no such thing as "sub-intermediate", I'm going to insist on labels that make sense. Future editing of the ontology requires consistency, otherwise we'll have to explain every internal organisations logic to other users.

> The label that @pbuttigieg proposed, which is going to be more understandable by external people, can be added as exact synonym.

Keep in mind, that SDGIO is external to the CGIAR and seeks to externalise it's strategy. The outward facing label should take precedence.

That being said, if there's a special label needed by an organisation, it can either be added as a synonym or as its own AP. For example, we have "UNEP preferred label" so that UN Environment's systems can display a label that they prefer. 

Practically speaking, systems should be focused on the IRIs and not the labels anyway.

Reply:
I am going to resolve this conversation.
Summary of the issue: the phrase "sub-intermediate development outcome" is hardly understandable by a user external to CGIAR.
Solution: I modified the label according to Pier Luigi's suggestion.